### DISCUSSION TITLE
Migrating to PlayIntegrity from SafetyNet for Firebase Appcheck

### DISCUSSION THREAD

**Discussion by fidodido75** [invertase/react-native-firebase#6980] [Q&A] [answered]
Our app is currently on ` react-native 0.67.4` with `react-native-firebase/app-check": "8.15.0-beta"`. As SafetyNetAttestation API is deprecated on Android, we need to migrate to Play Integrity API.
Q1: Is it possible to migrate to PlayIntegrity of this version of (`react-native-firebase/app-check": "8.15.0-beta"`) ?
Q2: If I were to upgrade to the latest version of `react-native-firebase/app-check": "17.3.0"`. Would it be compatible with ` react-native 0.67.4`? 
If the above two is not possible, I think I would be only left with the option of initially upgrading the react-native version and then upgrading the version of firebase.
Any advise would be highly appreciated.

**Comment by mikehardy** [comment, marked as answer]
Hi there!

Q1 - no, there is no way. You'll need that latest release to access the PlayIntegrity provider
Q2 - it should be compatible with 0.67.x - I don't recall major differences in react-native since then that we had to adapt to. Always better to be current, but I do not *think* there will be problems
